v0.0.1/admin-schema.json

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
general question: is there anything in this file that indicates "required" fields vs. optional?

Reply:
I saw the discussions in the slack where @elray1 explain where required v optional is defined in the schema but docson itself does not highlight this in the preview which is annoying. We might be able to hack something together in the widget's js to do this though.